docs/en/integrations/index.mdx

```diff
@@ -102,6 +102,7 @@ We are actively compiling this list of ClickHouse integrations below, so it's no
 |Amazon Kinesis|<KinesisSVG style={{width: '3rem', height: '3rem'}} />|Data ingestion|Integration with Amazon Kinesis.|[Documentation](/docs/en/integrations/clickpipes/kinesis/)|
 |Amazon MSK|<AmazonMskSVG style={{width: '3rem'}} />|Data ingestion|Integration with Amazon Managed Streaming for Apache Kafka (MSK).|[Documentation](/docs/en/integrations/kafka/cloud/amazon-msk/)|
 |Amazon S3|<S3SVG style={{width: '3rem', height: '3rem'}} />|Data ingestion|Import from, export to, and transform S3 data in flight with ClickHouse built-in S3 functions.|[Documentation](/docs/en/integrations/data-ingestion/s3/index.md)|
+|Amazon Glue|<img src={require('./images/logos/glue_logo.png').default} class="image" alt="Amazon Glue logo" style={{width: '3rem', 'background-color': 'transparent'}}/>|Data ingestion| Query ClickHouse over JDBC |[Documentation](/docs/en/integrations/glue)|
```

Comment:
```suggestion
|Amazon Glue|<img src={require('./images/logos/glue_logo.png').default} class="image" alt="Amazon Glue logo" style={{width: '3rem', 'background-color': 'transparent'}}/>|Data ingestion|Query ClickHouse over JDBC|[Documentation](/docs/en/integrations/glue)|
```

Reply:
@BentsiLeviav, just fix it and i already approved